src/SprykerEco/Yves/Adyen/Handler/Notification/Mapper/AdyenNotificationMapper.php

```diff
@@ -58,7 +58,7 @@ public function mapRequestToNotificationsTransfer(Request $request): AdyenNotifi
     }
 
     /**
-     * @param string[] $response
+     * @param array<string> $response
```

Comment:
Could we specify more precisely like `array<string, string>`?

Reply:
Fixed.